### ISSUE TITLE
Error message when creating server.

### ISSUE THREAD

**Issue by Qasimmodo** [Pterodactyl/Panel#476]
```
bunyan: ERROR: Exception thrown from the "res" Bunyan serializer. This should never happen. This is a bug in that serializer function.
RangeError: Maximum call stack size exceeded
    at ServerResponse.getHeaders (/srv/daemon/node_modules/restify/lib/response.js:176:52)
    at ServerResponse.get (_http_outgoing.js:130:17)
    at ServerResponse.getHeaders (/srv/daemon/node_modules/restify/lib/response.js:177:17)
    at ServerResponse.get (_http_outgoing.js:130:17)
    at ServerResponse.getHeaders (/srv/daemon/node_modules/restify/lib/response.js:177:17)
    at ServerResponse.get (_http_outgoing.js:130:17)
    at ServerResponse.getHeaders (/srv/daemon/node_modules/restify/lib/response.js:177:17)
    at ServerResponse.get (_http_outgoing.js:130:17)
    at ServerResponse.getHeaders (/srv/daemon/node_modules/restify/lib/response.js:177:17)
    at ServerResponse.get (_http_outgoing.js:130:17)
    at ServerResponse.getHeaders (/srv/daemon/node_modules/restify/lib/response.js:177:17)
```
This is in the daemon.

**Comment by schrej** [comment]
What service?

Please try deleting node_modules/ and run `npm install` again

**Comment by DaneEveritt** [comment]
What version of node JS? This error is has nothing to do with any of our code.

**Comment by Qasimmodo** [comment]
Resolved, It was because I was using NodeJS 8.0.0, went back to 6.0.0 as the docs suggested and works perfectly. Thank you.
